middleware/pprof/setup.go

```diff
@@ -1,13 +1,18 @@
 package pprof
 
 import (
+	"net"
 	"sync"
 
 	"github.com/coredns/coredns/middleware"
 
 	"github.com/mholt/caddy"
 )
 
+const (
+	default_addr = "localhost:6053"
+)
+
```

Comment:
you can drop the `(` and `)` as there is only 1 const defined. Also `default_addr` -> `defaultAddr`